hagglehaul.Server/EmailViews/AcceptedBidEmail.cshtml

```diff
@@ -1,24 +1,62 @@
 @model hagglehaul.Server.EmailViews.AcceptedBidEmail
 
-@{
-    Layout = null;
-}
-
 <!DOCTYPE html>
-
 <html lang="en">
 <head>
-    <title>Hagglehaul Driver Management: Your bid has been accepted</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Accepted Bid Notification</title>
     <style>
-        .blue-accent {
-            color: #126391;
+        body {
+            font-family: Arial, sans-serif;
+            background-color: #f4f4f4;
+            padding: 20px;
+        }
+        .container {
+            background-color: #fff;
+            border-radius: 8px;
+            box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+            padding: 20px;
+            margin: 20px auto;
+            max-width: 600px;
+        }
+        h1 {
+            color: #333;
+            text-align: center;
+        }
+        p {
+            color: #666;
+            line-height: 1.6;
+        }
+        .footer {
+            margin-top: 20px;
+            text-align: center;
+            color: #999;
         }
     </style>
 </head>
 <body>
-<div>
-    <h1>Accepted bid email template</h1>
-    <p class="blue-accent">@Model.ExampleProperty</p>
-</div>
+    <div class="container">
+        <h1>Accepted Bid Notification</h1>
+        <p>Dear @Model.DriverName,</p>
+        <p>Your bid has been accepted. Please review the details below:</p>
+        <a href="https://hagglehaul.azurewebsites.net" target="_blank">Click here to manage your trips</a>
```

Comment:
Move below everything else, above "do not reply." Change back to the click for more details message (drivers do not manage trips, so this doesn't make sense to say to everyone).